waspc/data/Generator/templates/react-app/scripts/validate-env.mjs

```diff
@@ -0,0 +1,7 @@
+// NOTE: validators are copied from shared templates
```

Comment:
Since we are outside of the `src` dir and working with plain Node.js scripts, I needed to use the `mjs` extensison to indicate that this file is in `ES module` format.